### PR TITLE
feat: Adds content to active detour panel

### PR DESCRIPTION
**Pull request by hannahpurcell**
Asana ticket: https://app.asana.com/0/1205385723132845/1208153509160250

My last commit is optional! If this rearrangement is not valuable, I'm happy to revert. I just wanted to make a few pieces of the detour panels reusable.

Another idea for another PR: putting all detour panels in a subfolder together

### CODE REVIEW

File: assets/src/components/detours/diversionPage.tsx [hunk 1]
```diff
@@ -245,6 +245,17 @@ export const DiversionPage = ({
...
+              connectionPoints={[
+                connectionPoints?.start?.name ?? "N/A",
+                connectionPoints?.end?.name ?? "N/A",
+              ]}
+              missedStops={missedStops}
+              routeName={routeName ?? "??"}
+              routeDescription={routeDescription ?? "??"}
+              routeOrigin={routeOrigin ?? "??"}
+              routeDirection={routeDirection ?? "??"}
+              onNavigateBack={() => onConfirmClose}
```
Comment (joshlarson) [inline assets/src/components/detours/diversionPage.tsx:258]:
```suggestion
              onNavigateBack={onClose}
```

In order to make the back button behave the same as the close button.

**Review by joshlarson — CHANGES_REQUESTED** [review]
I'm a big fan of extracting the reusable components, and I like the way this shook out.

Found a bug with the `<- Back` button, and want to take another look in a little bit, but overall I like it!

File: assets/src/components/detours/diversionPage.tsx (hunk 1 above)
Comment (hannahpurcell) [reply assets/src/components/detours/diversionPage.tsx:258]: We don't want it to behave the same as the close button. For an active detour, we don't need the confirm close modal to appear

Comment (joshlarson) [reply assets/src/components/detours/diversionPage.tsx:258]: Oh yeah I guess the confirm dialog isn't really necessary from the active state.

In that case:
```suggestion
              onNavigateBack={onConfirmClose}
```

**Review by joshlarson — APPROVED** [review]

**Pull request merged**